fixtures/ayra/ero-micro-beam-fx.json

```diff
@@ -0,0 +1,229 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "ERO Micro Beam FX",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["M. Peters"],
+    "createDate": "2024-06-21",
+    "lastModifyDate": "2024-06-21"
+  },
+  "links": {
+    "manual": [
+      "https://www.bax-shop.nl/index3.php?option=com_download_files&product_id=514480&download_id=170502"
+    ],
+    "productPage": [
+      "https://www.bax-shop.nl/moving-head/ayra-ero-micro-beam-fx"
+    ],
+    "video": [
+      "https://www.bax-shop.nl/moving-head/ayra-ero-micro-beam-fx#video"
+    ]
+  },
+  "physical": {
+    "dimensions": [200, 140, 240],
+    "weight": 2.6,
+    "power": 35,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "1x 15W RGBW multichip LED en 24x 5050RGB SMD LEDs"
+    },
+    "lens": {
+      "degreesMinMax": [5, 25]
```

Comment:
The manual says "Beam angle: 4 Degree beam, 25 Degree ring", so this is technically (mostly) right, but I'm not sure if we want to encode it differently.  It's not really what I think of when I hear a lens has a min/max of 5-25 degrees.